provider/blob/src/export/blob_exporter.rs

```diff
@@ -43,13 +51,32 @@ impl core::fmt::Debug for BlobExporter<'_> {
 }
 
 impl<'w> BlobExporter<'w> {
-    /// Create a [`BlobExporter`] that writes to the given I/O stream.
+    /// Creates a version 1 [`BlobExporter`] that writes to the given I/O stream.
+    ///
+    /// Version 1 is needed if the blob may be consumed by ICU4X versions 1.0 through 1.3. If
+    /// targeting only ICU4X 1.4 and above, see [BlobExporter::new_v2_with_sink()].
     pub fn new_with_sink(sink: Box<dyn std::io::Write + Sync + 'w>) -> Self {
         Self {
-            resources: Mutex::new(Vec::new()),
+            resources: Mutex::new(BTreeMap::new()),
```

Comment:
nit: change to `Default::default()` where possible

Reply:
done